### PR TITLE
Banner for New version

### PR DESCRIPTION
**Pull request by kadmil**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -38,6 +38,9 @@
   <meta name="msapplication-config" content="/img/win81/browserconfig.xml" />
 </head>
 <body class="transition-active transition-static">
+  <div class="annotation-title">
+    Вы зашли на старую версию интерфейса Фрифида. Она пока работает, но всё  интересное добавляется только в новой. <a href='https://beta.freefeed.net'>[Перейти к новой версии]</a>
```
Comment (berkus) [inline public/index.html]: No need for square brackets here.

Comment (berkus) [inline public/index.html]: Href should point to the same page but with different host (e.g. on `freefeed.net/user/POSTUUID` the url should be `beta.freefeed.net/user/POSTUUID`) to allow seamless switch. See how redirect to `m.freefeed.net` works on ember loader page.

File: public/index.html [hunk 2]
```diff
@@ -38,6 +38,9 @@
   <meta name="msapplication-config" content="/img/win81/browserconfig.xml" />
```
Comment (berkus) [inline public/index.html:38]: When did we merge the windows tiles stuff?

File: public/index.html (hunk 1 above)
Comment (berkus) [inline public/index.html]: UI text should be in English attn @urbansheep

File: public/js/app/router.js [hunk 3]
```diff
@@ -50,4 +50,11 @@ define(["app/app"], function(App) {
         });
     }.on('didTransition')
   });
+
+  App.Router.reopen({
+    changeBanner: function() {
+      var bannerLink = document.querySelectorAll('.annotation-title a')[0]
+      bannerLink.href = 'https://beta.' + location.hostname + location.pathname
```
Comment (clbn) [inline public/js/app/router.js]: We should add `location.search` here, otherwise `?offset=30` will be lost. JUST LOST!

**Comment by clbn** [comment]
:+1:

**Pull request merged**